src/en/charms-deploying.md

```diff
@@ -237,7 +259,7 @@ juju retry-provisioning 3 27 5
 
 ## Considerations
 
-Charms are running without any separation, so its entirely possible for Charms
+Charms are running without any separation, so its entirely possible for charms
 to stomp all over each others configuration files and try to bind to the same
```

Comment:
"...to stomp all over each other's configuration files..."?
First, the obvious missing apostrophe.
Second, is this the phrasing we want to use? It feels like it has a negative connotation to me, and we probably want to keep the editorializing internal and project our best face to customers.